### PR TITLE
[risk=no][no ticket] Convert 2 FA pro icons to free

### PR DESCRIPTION
**Pull request by jmthibault79**
Circle and Repeat have Free equivalents, so we can use those.  After this PR, only 4 Pro icons remain: https://docs.google.com/document/d/1X9KX_hegNrMXf-XjwZWv1-qimS-tg2xdd5q61UljWxU/edit

Also a bit of other icon cleanup.

Tested by running locally.

Pro Circle
Screenshot 2023-10-13 at 12 09 25 PM

Free Circle
Screenshot 2023-10-13 at 12 09 44 PM

Pro Repeat
Screenshot 2023-10-13 at 11 59 28 AM

Free Repeat
Screenshot 2023-10-13 at 11 59 59 AM

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have manually tested this change and my testing process is described above.
- [ ] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [x] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [ ] None of the above apply to this change.

### CODE REVIEW

File: ui/src/app/components/icons.tsx [hunk 1]
```diff
@@ -9,15 +10,13 @@ import {
   faExclamationTriangle,
   faLongArrowAltRight,
   faMinusCircle,
-  faTimes,
```
Comment (jmthibault79) [inline ui/src/app/components/icons.tsx:12]: not used

File: ui/src/app/components/icons.tsx [hunk 2]
```diff
@@ -9,15 +10,13 @@ import {
...
+  faRepeat,
 } from '@fortawesome/free-solid-svg-icons';
-import { faCircle } from '@fortawesome/pro-regular-svg-icons';
 import { faCircleEllipsisVertical } from '@fortawesome/pro-regular-svg-icons';
-import { faAlarmExclamation, faRepeat } from '@fortawesome/pro-solid-svg-icons';
+import { faAlarmExclamation } from '@fortawesome/pro-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import colors, { addOpacity } from 'app/styles/colors';
-import arrow from 'assets/icons/arrow-left-regular.svg';
```
Comment (jmthibault79) [inline ui/src/app/components/icons.tsx:20]: not used here (used in other files though)

File: ui/src/app/components/icons.tsx [hunk 3]
```diff
@@ -188,16 +187,6 @@ const Icon = ({ shape, size, style, color, ...props }) => {
   );
 };
 
-export const withCircleBackground =
```
Comment (jmthibault79) [inline ui/src/app/components/icons.tsx:191]: not used

**Review by freemabd — APPROVED** [review]
LGTM

**Pull request merged**